docs/editors/new-editor.md

```diff
@@ -88,16 +88,23 @@ Possible values:
 
 ### `-Dmetals.file-watcher`
 
+This option is no longer used by Metals.
+
+### `-Dmetals.directory-glob`
```

Comment:
Bikeshedding: IMO the name is not very clear. How about something like `glob-syntax` or `watch-pattern-type` ?